metricflow/cli/sample_models/countries.yaml

```diff
@@ -1,8 +1,6 @@
 semantic_model:
   name: countries
   description: Regions mapped to countries.
-  defaults:
```

Comment:
This should work - what's the context on removing it?

Reply:
This model has no dimensions so its failing the validations